control-plane/Dockerfile

```diff
@@ -42,6 +43,7 @@ RUN addgroup ${BIN_NAME} && \
     adduser -S -G ${BIN_NAME} 100
 
 COPY pkg/bin/linux_${ARCH}/${BIN_NAME} /bin
+COPY cni/pkg/bin/linux_${ARCH}/${CNI_BIN_NAME} /bin
```

Comment:
The plugin is under control-plane but the binary ends up in another location. I didn't want to mess with build-local.sh too much and move files around, to say ./control-plane/bin, so I left the binary in the cni directory.